### PR TITLE
MMT 1472 - added active record session store.

### PR DESCRIPTION
**Pull request by cgokey**
Session store now using active record.

### CODE REVIEW

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_cmr-metadata-review_session'
+Rails.application.config.session_store :active_record_store, :key => '_my_app_session'
```
Comment (cgokey) [inline config/initializers/session_store.rb]: Use the original key

**Comment by cgokey** [comment]
use the whenever gem to trim sessions.

File: config/initializers/session_store.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_cmr-metadata-review_session'
+Rails.application.config.session_store :active_record_store, :key => '_cmr-metadata-review_session'
```
Comment (charleshuang80) [inline config/initializers/session_store.rb]: you should keep the syntax as `key:` (they're equivalent but this one is preferred if it is possible)

File: config/schedule.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Use this file to easily define all of your cron jobs.
+#
+set :output, "/tmp/cron_log.log"
+every 1.day, at: '4:00 am' do
+  rake "db:sessions:trim"
```
Comment (charleshuang80) [inline config/schedule.rb]: This rake task and the cron job don't output anything if it runs successfully. So for MMT Hoan created a different rake task that invokes this one and also adds something to the logs so we know it was run successfully. We probably want to do the same thing here.

File: config/schedule.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Use this file to easily define all of your cron jobs.
+#
+set :output, "/tmp/cron_log.log"
```
Comment (cgokey) [inline config/schedule.rb]: Just noticed, I thought I took out the set output, I probably shouldn't write to /tmp, there is another ticket to in the queue for handling writing logs to splunk, but is still open.   I'll look at what Hoan has done interim though.

**Pull request merged**